### PR TITLE
Add views for browsing collections history

### PR DESCRIPTION
**Pull request by n1k0**
Feedback=? @leplatrem

### CODE REVIEW

File: scripts/components/CollectionHistory.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      record_id,
+      user_id,
+    } = entry;
+    return (
+      <tbody>
+        <tr>
+          <td>{date}</td>
+          <td>{action}</td>
+          <td>{resource_name}</td>
+          <td>{resource_name === "collection" ? collection_id : record_id}</td>
```
Comment (leplatrem) [inline scripts/components/CollectionHistory.js]: `target.data.id`

File: scripts/components/CollectionHistory.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          <td>{user_id}</td>
+          <td className="text-center">
+            <a href="." className="btn btn-xs btn-default"
+               onClick={this.toggle}>
```
Comment (leplatrem) [inline scripts/components/CollectionHistory.js]: toolTip: show history entry details

File: scripts/components/CollectionHistory.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+              <i className={`glyphicon glyphicon-eye-${open ? "close" : "open"}`} />
```
Comment (leplatrem) [inline scripts/components/CollectionHistory.js]: question: use triangles ?

File: scripts/components/CollectionHistory.js (hunk 1 above)
Comment (leplatrem) [inline scripts/components/CollectionHistory.js]: idea: add link on id to collection/record edition

File: scripts/components/Sidebar.js [hunk 4]
```diff
@@ -7,31 +7,85 @@ function activeIfPathname(location, pathname) {
...
+
+  return (
+    <div className={classes}>
+      <i className="glyphicon glyphicon-align-justify"/>
+      {
+        currentPath === listPath ? id : <Link to={listPath}>{id}</Link>
+      }
+      <GearMenu>
+        <Link to={listPath}>
+          <i className="glyphicon glyphicon-search" />Browse records
```
Comment (leplatrem) [inline scripts/components/Sidebar.js]: icon: align-justify ?

File: scripts/sagas/collection.js [hunk 5]
```diff
@@ -45,6 +45,23 @@ export function* listRecords(getState, action) {
...
 
+export function* listHistory(getState, action) {
+  const {bid, cid} = action;
+  try {
+    const bucket = getBucket(bid);
+    const {data} = yield call([bucket, bucket.listHistory], {
+      sort: "-date",
+      filters: {
+        collection_id: cid,
+        // resource_name: "record",
```
Comment (leplatrem) [inline scripts/sagas/collection.js]: leftover

File: scripts/sagas/collection.js [hunk 6]
```diff
@@ -45,6 +45,23 @@ export function* listRecords(getState, action) {
...
+      }
```
Comment (leplatrem) [inline scripts/sagas/collection.js]: note for later: `?limit=` ?

**Comment by leplatrem** [comment]
r+ :)

File: scripts/components/CollectionHistory.js (hunk 3 above)
Comment (n1k0) [inline scripts/components/CollectionHistory.js]: Couldn't find anything working :/

**Pull request merged**